src/interceptors/ClientRequest/NodeClientRequest.test.ts

```diff
@@ -151,6 +148,10 @@ test('emits the ENOTFOUND error connecting to a non-existing hostname given no m
     done()
   })
 
+  request.on('response', () => {
```

Comment:
Looks like a debugging leftover.

Reply:
it is executed on my local where this test fails

Reply:
Could you post the test runner's result when this test fails? I wonder what exactly makes it fail: it actually connects to a non-existing-url.com?

Reply:
<img width="518" alt="image" src="https://user-images.githubusercontent.com/22871609/177044839-a8194433-358f-497a-8412-a8ef2c9477c4.png">

Reply:
Here's what I'm getting:
```sh
% ping non-existing-url.com
ping: cannot resolve non-existing-url.com: Unknown host
```

Perhaps you have some custom host configuration on your machine? 

In any case, try setting a different bust still valid URL and see if it results in the same test failure maybe? It doesn't really matter what URL we try to connect here, only that the connection fails. It can be `<uuid>.com`.